pylint_django/checkers/migrations.py

```diff
@@ -156,7 +156,7 @@ def func(apps, schema_editor):
     return is_migrations_module(node.parent)
 
 
-def load_configuration(linter):
+def load_configuration(linter):  # TODO this is redundant and can be  removed
```

Comment:
+1 for adding the TODOs as separate issues so we don't forget about them. Interestingly enough the pylint test job in Travis should be catching this.

Reply:
Agreed - I had initially not bothered as I had planned to immediately address them in a new PR but I got distracted / didn't get time. Therefore I've made https://github.com/PyCQA/pylint-django/issues/289 and https://github.com/PyCQA/pylint-django/issues/288